recipes/ocp/generate-bindings.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+set -euo pipefail
+
+PLATFORM=$(uname)
+
+declare -a EXTRA_INCLUDES
+
+if [[ ${PLATFORM} = 'Darwin' ]]; then
+  CLANGLIB=${PREFIX}/lib/libclang.dylib
```

Comment:
a lot of this is taken care of by the compiler activation packages you get "for free" with `compiler('cxx')`.

Reply:
I thought that, but the pywrap process seems not to be as aware of environment variables as something like CMake.

This is turning out to be quite challenging, so I've asked the OCP authors if they can make the binding code a release of its own. Compiling that code is (I think) much more straightforward.

Reply:
What is actually challenging here?

How is the location of `libclang` communicated by setting `compiler('CXX')`? Why would one assume that `compiler('CXX')` provides `libclang` at all?

Reply:
This is outdated as I am no longer attempting to generate the bindings in the recipe. The current challenges are unrelated to this particular issue. I will resolve this conversation to prevent future confusion.